lib/presentation/language_tool_text_field.dart

```diff
@@ -42,6 +42,8 @@ class LanguageToolTextField extends StatefulWidget {
     this.maxLines = 1,
     this.minLines,
     this.expands = false,
+		this.textAlign = TextAlign.right,
+		this.textDirection = TextDirection.rtl,
```

Comment:
Manually specifying directionality makes sense for apps that are expected to support multiple languages at once -- but even then I'd consider creating scoped [Directionality](https://api.flutter.dev/flutter/widgets/Directionality-class.html) contexts.

Nevertheless, it's still sensible to have these arguments available.

But if we want them to be part of the main package -- I suggest we make them nullable and optional, `null` as default.
```suggestion
		this.textAlign,
		this.textDirection,
```

Reply:
Correction: we won't be able to make textAlign nullable -- instead we should copy the default value from [TextField constructor](https://api.flutter.dev/flutter/material/TextField/TextField.html).

```suggestion
		this.textAlign = TextAlign.start,
		this.textDirection,
```

Reply:
Ok, this is better!